README.md

```diff
@@ -21,11 +21,12 @@ for more details about the various versions of the driver.
 
 ## Resources
 
-* [MongoDB C++ Driver Quickstart](https://mongodb.github.io/mongo-cxx-driver/mongocxx-v3/tutorial/)
-* [MongoDB C++ Driver Manual](https://mongodb.github.io/mongo-cxx-driver/)
-* [MongoDB C++ Driver API Documentation](https://mongodb.github.io/mongo-cxx-driver/api/mongocxx-v3)
-* [MongoDB C++ Driver Contribution guidelines](https://mongodb.github.io/mongo-cxx-driver/contributing/)
-* [MongoDB Database Manual](http://docs.mongodb.com/manual/)
+* [MongoDB C++ Driver Installation](http://mongocxx.org/mongocxx-v3/installation/)
+* [MongoDB C++ Driver Quickstart](http://mongocxx.org/mongocxx-v3/tutorial/)
+* [MongoDB C++ Driver Manual](http://mongocxx.org)
```

Comment:
Also use the mongocxx.org URLs instead of the mongodb.github.io URLs (which redirect). Note, this changes the link from "https" to "http", but the current redirects go to http anyway.

It'd be good to switch this to https when the docs site supports it.